src/game_player.cpp

```diff
@@ -673,6 +673,13 @@ bool Game_Player::GetOffVehicle() {
 	return true;
 }
 
+bool Game_Player::IsStopping() const {
+	// Issue #1268 (player moving too early on a moveroute when using airship) was because MoveRoutes use this.
```

Comment:
How about "// Prevent MoveRoute execution while the airship is ascending"

Reply:
*nods* Will add that now.